app/src/main/java/org/mozilla/fenix/components/Services.kt

```diff
@@ -42,6 +47,17 @@ class Services(
         }
     }
 
+    val appLinksInterceptor by lazy {
+        AppLinksInterceptor(
+            context,
+            interceptLinkClicks = true,
+            launchInApp = {
+                context.settings().preferences.getBoolean(
+                    context.getPreferenceKey(R.string.pref_key_open_links_in_external_app), false)
+            }
+        )
+    }
```

Comment:
Is that a `service`? 🤔

Reply:
Not really, but I don't know a better place to put it.  The other interceptor current in use is the `FirefoxAccountsAuthFeature` which is just above this.